towhee/dag/graph_repr.py

```diff
@@ -21,16 +23,25 @@
 
 
 class GraphRepr(BaseRepr):
-    """This class presents a complete representation of a graph and its individual
-    subcomponents, including Operators, Dataframes, and Variables. Graph representations
-    are used during execution to load functions and pass data to the correct operators.
-    """
+    """This class presents a complete representation of a graph.
 
-    def __init__(self, name: str):
-        # TODO(Chiiizzzy)
+    A graph contains individual subcomponents, including Operators, Dataframes, and
+    Variables. Graph representations are used during execution to load functions and
+    pass data to the correct operators.
+    """
+    def __init__(self, name: str, file_or_url: str = None):
         super().__init__(name)
         self._operators = {}
         self._dataframes = {}
+        # if `file` is a loacl file
+        if file_or_url and os.path.isfile(file_or_url):
+            with open(file_or_url, 'r', encoding='utf-8') as f:
+                self._src = yaml.safe_load(f)
+            self.from_yaml(self._src)
+        # if `file` is a remote url
+        elif file_or_url:
+            self._src = requests.get(file_or_url).content.decode('utf-8')
```

Comment:
Let's add a test case for remote file upload as well. As a source, we can create a gist for `towhee-io`.